Sources/ComposableArchitecture/SwiftUI/IdentifiedArray.swift

```diff
@@ -14,7 +14,7 @@ import Foundation
 ///
 /// This domain can be pulled back to a larger domain with the `forEach` method:
 ///
-///     struct AppState { var counters = IdentifiedArray<Int>(id: \.self) }
+///     struct AppState { var counters = IdentifiedArray<UUID, CounterState>() }
```

Comment:
Good catch! I think we could even use the type alias:
```suggestion
///     struct AppState { var counters = IdentifiedArrayOf<CounterState>() }
```